docs/source/package_orderers.rst

```diff
@@ -0,0 +1,86 @@
+================
+Package Orderers
+================
+
+Rez's default version resolution algorithm will always sort by the latest alphanumeric
```

Comment:
Should we reference https://rez.readthedocs.io/en/stable/basic_concepts.html#versions?